Documentation/README_LostTrackingService.md

```diff
@@ -0,0 +1,10 @@
+# Lost Tracking Visualization
+
+![Lost Tracking](../Documentation/Images/LostTracking/LostTrackingVisualization.jpg)
+
+Lost Tracking Extension Service provides HoloLens shell style animated visual for the lost tracking state.
```

Comment:
Is there a way to add lost tracking service via code? can you please add documentation?